src/admin/content-block/content-block.gql.ts

```diff
@@ -14,7 +14,12 @@ export const GET_CONTENT_BLOCKS_BY_CONTENT_ID = gql`
 
 export const INSERT_CONTENT_BLOCKS = gql`
 	mutation insertContentBlocks($contentBlocks: [app_content_blocks_insert_input!]!) {
-		affected_rows
+		insert_app_content_blocks(objects: $contentBlocks) {
+			affected_rows
```

Comment:
ik denk dat enkel de returning nodig is om de query uit te voeren, de affected_rows kan je weg laten

Reply:
Fixed